ls1intum/Artemis
# Fix/lecture units improvements

### Checklist

### Motivation and Context



### Description

- updated the style of the lecture units to a cleaner and more modern look
- text units should now open in a new tab instead of a popup when the button is clicked
- removed debounce-click directive


### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create, Update, Delete Lecture Units of all types as a instructor, view them as a student and see that everything still works

### Test Coverage
No changes in code just fixed and styling changes

### Screenshots


Before:
![image](https://user-images.githubusercontent.com/29718932/100114045-4a707c00-2e71-11eb-8aed-d219937472b6.png)
![image](https://user-images.githubusercontent.com/29718932/100114156-683de100-2e71-11eb-9e07-6c527e671301.png)

After:
![image](https://user-images.githubusercontent.com/29718932/100113877-1bf2a100-2e71-11eb-9027-fe48bda5d4c2.png)
![image](https://user-images.githubusercontent.com/29718932/100114305-902d4480-2e71-11eb-92af-c08b4db9a376.png)
